src/libs/kata-types/src/initdata.rs

```diff
@@ -206,6 +206,11 @@ pub fn calculate_initdata_digest(
 /// The argument `initda_annotation` is a Standard base64 encoded string containing a TOML formatted content.
 /// This function decodes the base64 string, parses the TOML content into an InitData structure.
 pub fn add_hypervisor_initdata_overrides(initda_annotation: &str) -> Result<String> {
+    // If the initdata is empty, return an empty string
+    if initda_annotation.is_empty() {
+        return Ok("".to_string());
```

Comment:
Let's also add a logging or something [here](https://github.com/kata-containers/kata-containers/blob/main/src/runtime-rs/crates/runtimes/virt_container/src/sandbox.rs#L441-L442) to show that no initdata is given. This would be help for debugging